.github/workflows/examples-bouffalolab.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
     if: github.actor != 'restyled-io[bot]'
 
     container:
-      image: ghcr.io/project-chip/chip-build-bouffalolab:54
+      image: ghcr.io/project-chip/chip-build-bouffalolab:67
```

Comment:
This change has to be made in a separate PR from the dockerfile update: we update the dockerfile after the commit, so there ia few hours lag until they are available.

Reply:
Updated to only contain chagnes for Docker image build.
Rest changes will be in another PR when Docker image is accessible.